src/setupProxy.js

```diff
@@ -0,0 +1,9 @@
+const { createProxyMiddleware } = require("http-proxy-middleware");
+module.exports = function (app) {
+  app.use(
+    "/__",
```

Comment:
The script tags I added all point to `/__`.
this path is use by firebase hosting to initialize everything, but we need to tell react to proxy this path to our local host when developing so we still get the hot loader